### ISSUE TITLE
[Shattrath City Issues - Harbinger Haronem is not scripted, thus no Aldor-war event]

### ISSUE THREAD

**Issue by Paradigmxd** [chromiecraft/chromiecraft#5471]
### What client do you play on?

enUS

### Faction

Both

### Content Phase:

70

### Current Behaviour

Harbinger Haronem is stationary and only has his stock quote.

### Expected Blizzlike Behaviour

Harbinger Haronem has a unique and funny event with his Aldor Marksmen troops who he prepares for the war ^_^.

Here is some info

Dialogue
[Harbinger Haronem](https://wowpedia.fandom.com/wiki/Harbinger_Haronem) says: Soup's on, men! Eat up. We've got a long day ahead.
Harbinger Haronem says: Fall in men! It's time to drill!
Harbinger Haronem says: Alright now, let's get those rifles sighted in...
[Aldor Marksman](https://wowpedia.fandom.com/wiki/Aldor_Marksman) says: ...at least we know my rifle is sighted properly.
Harbinger Haronem says: Soldier...
Harbinger Haronem says: You DO know that [marksmen](https://wowpedia.fandom.com/wiki/Marksman) are issued blanks for practice drills, right?
Harbinger Haronem says: Clearly, this group is not ready for weapons exercises.
Harbinger Haronem says: Fall in for some marching practice! Let's see if we can manage to walk without suffering any casualties...
Harbinger Haronem says: That was some pretty fancy prancin' ladies!
Harbinger Haronem says: Now let's give shootin' another try...
Harbinger Haronem says: Get into formation and get those guns sighted in!
Harbinger Haronem says: Carefully now, we don't need any more mista- (this is where his marksmen point the gun on him and pretend to shoot him and he emotes like /fear)
Harbinger Haronem says: Oh, that was cute...
Harbinger Haronem says: We'll be marching all day if I see any more of that!
Harbinger Haronem says: In fact, let's see some more of that fancy walkin'... Move it!
Harbinger Haronem says: Again, very nice strutting ladies.
Harbinger Haronem says: If we can't teach you to shoot, you'll have a career in the cantina to fall back on.
Harbinger Haronem says: What's this? You've shifted into an incorrect formation!
Harbinger Haronem says: You are only to use the staggered formation just before atta-
Harbinger Haronem says: Enough messing around! There's a war going on here!
Harbinger Haronem says: At ease, men.

### Source

https://wowpedia.fandom.com/wiki/Losing_Gracefully#Notes

https://www.wowhead.com/wotlk/npc=19475/harbinger-haronem#comments

I recommend reading the comments to understand whats happening, but its pretty much him walking around his marksmen groups and using different dialogues

He also gets 'pretend shot' by his troops which really made this Shattrath Event quite funny to watch!

### Steps to reproduce the problem

1. Teleport to Harbinger Haronem NPC ID 19475 in Shattrath City next to Scryers' bank
2. Observe if he says anything but It's coming along, [Wafflefry]. Haven't you got a product demonstration to do or something?
3. Confirm he has no event scripted.
4. End

### AC rev. hash/commit

https://github.com/chromiecraft/azerothcore-wotlk/commit/ce6ca597b54e64f328cacddae5f003cdc41a343d

### Operating system

Ubuntu 20.04

### Modules

- [mod-ah-bot](https://github.com/azerothcore/mod-ah-bot)
- [mod-bg-item-reward](https://github.com/azerothcore/mod-bg-item-reward)
- [mod-BreakingNewsOverride](https://github.com/AnchyDev/BreakingNewsOverride)
- [mod-cfbg](https://github.com/azerothcore/mod-cfbg)
- [mod-chat-transmitter](https://github.com/azerothcore/mod-chat-transmitter)
- [mod-cta-switch](https://github.com/azerothcore/mod-cta-switch)
- [mod-desertion-warnings](https://github.com/azerothcore/mod-desertion-warnings)
- [mod-duel-reset](https://github.com/azerothcore/mod-duel-reset)
- [mod-eluna](https://github.com/azerothcore/mod-eluna)
- [mod-instanced-worldbosses](https://github.com/nyeriah/mod-instanced-worldbosses)
- [mod-ip-tracker](https://github.com/azerothcore/mod-ip-tracker)
- [mod-low-level-arena](https://github.com/azerothcore/mod-low-level-arena)
- [mod-low-level-rbg](https://github.com/azerothcore/mod-low-level-rbg)
- [mod-multi-client-check](https://github.com/azerothcore/mod-multi-client-check)
- [mod-progression-system](https://github.com/azerothcore/mod-progression-system)
- [mod-pvp-titles](https://github.com/azerothcore/mod-pvp-titles)
- [mod-pvpstats-announcer](https://github.com/azerothcore/mod-pvpstats-announcer)
- [mod-queue-list-cache](https://github.com/azerothcore/mod-queue-list-cache)
- [mod-rdf-expansion](https://github.com/azerothcore/mod-rdf-expansion)
- [mod-transmog](https://github.com/azerothcore/mod-transmog)
- [mod-weekend-xp](https://github.com/azerothcore/mod-weekend-xp)
- [mod-zone-difficulty](https://github.com/azerothcore/mod-zone-difficulty)
- [lua-carbon-copy](https://github.com/55Honey/Acore_CarbonCopy)
- [lua-event-scripts](https://github.com/55Honey/Acore_eventScripts)
- [lua-exchange-npc](https://github.com/55Honey/Acore_ExchangeNpc)
- [lua-level-up-reward](https://github.com/55Honey/Acore_LevelUpReward)
- [lua-recruit-a-friend](https://github.com/55Honey/Acore_RecruitAFriend)
- [lua-send-and-bind](https://github.com/55Honey/Acore_SendAndBind)
- [lua-temp-announcements](https://github.com/55Honey/Acore_TempAnnouncements)
- [lua-zonecheck](https://github.com/55Honey/acore_Zonecheck)

### Customizations

None

### Server

ChromieCraft

**Comment by Santaria94** [comment]
This event is related to the removed quest "Losing Gracefully" in patch 2.4.0. 
It was also not available on TBC Classic. The only video that seems to be left from that time: https://youtu.be/vteHbRWZ0Ts
I checked on the PTR and I can confirm that this quest is not scripted. However, I don't see the point in moving this report to the main AzerothCore repo, since the quest has been removed.
Maybe someone from the CC team will want to implement this as part of the Progression mod.
